testsuite/unittests/test_hh_psc_alpha_gap.sli

```diff
@@ -85,6 +85,9 @@ neuron1 neuron2
 Connect
 vm neuron2     1.0 h Connect
 
+% The following call to GetConnections makes Simulate crash
+% or generate results containing NaNs, see #2614.
+<< >> GetConnections ; 
```

Comment:
I think that it is better to create a specific test for this case rather than inserting the `GetConnection` without relay using the returned connections.